docs/protodoc-markdown.tmpl

```diff
@@ -3,37 +1,57 @@
+{{if false}}
+<!--
+The default template can be found here: https://github.com/pseudomuto/protoc-gen-doc/blob/master/resources/markdown.tmpl
+
+Differences between this template and the default:
+* In the tables, backticks were added around the .Name values as well as the enum .Number values.
+* Changed all 'nobr .Description' into 'nobr .Description | replace "\n\n" "<br>"'.
+  Without either nobr or the replace, the line breaks in the proto comments break the markdown tables.
+  With just nobr, an empty comment line in the proto breaks the markdown tables.
+  With just the replace, it still messes up on multi-line comments. If the replace is just "\n" "<br>" then
+    the cell looks weird because there are line breaks in the middle of sentences.
+  By using 'nobr .Description | replace "\n\n" "<br>"', the <br> tags are put between comment paragraphs which
+    makes them more natural to read, and they're all put on a single line, which makes the tables happy.
+* Added this comment to the top in an 'if false' block so that it doesn't get included in the final markdown file.
+
+The comment below stating that this file is auto-generated, is not true for this template file.
+It is there so that the resulting markdown file has that comment at the top.
+-->
+{{end}}<!-- This file is auto-generated. Please do not modify it yourself. -->
```

Comment:
The comment above it explains that that comment is there so that the final result has it at the top. It's located where it is because if it's where you're suggesting, then the generated result has an empty line at the top of it.